### DISCUSSION TITLE
In graph view display changes of first changed file automatically

### DISCUSSION THREAD

**Discussion by wl2776** [sourcegit-scm/sourcegit#779] [Ideas]
My suggestion, in this layout:

![image](https://github.com/user-attachments/assets/f3d09906-9d01-465e-bd51-02142d60e2ba)

instead of writing "Select file to view changes", just select and display first changed file, like below

![image](https://github.com/user-attachments/assets/89ad24fa-1f7a-4bc6-8135-93cdd7f6ea8b)

**Comment by love-linger** [comment]
#754
